scripts/build-shotcut.sh

```diff
@@ -20,10 +20,13 @@ SOURCE_DIR="$INSTALL_DIR/src"
 ACTION_GET_COMPILE_INSTALL=1
 ACTION_GET_ONLY=0
 ACTION_COMPILE_INSTALL=1
+TAR_APP_PACKAGE=1
+KEEP_BUILD_FILES=0
 SOURCES_CLEAN=1
 INSTALL_AS_ROOT=0
 CREATE_STARTUP_SCRIPT=1
 ENABLE_FREI0R=1
+DISABLE_ALL_EXCEPT_SHOTCUT_AND_MLT=0
```

Comment:
Sure. That would work. But the logic remains: if SUBDIRS is not specified (empty) then figure it out. Else use the projects specified in SUBDIRS.